### DISCUSSION TITLE
Welcome to Gethugothemes Discussions!

### DISCUSSION THREAD

**Discussion by somratpro** [teamosis/gethugothemes-documentation#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by davidsneighbour** [comment]
Hi there, Patrick here :) Let's use Discussions too to collect ideas for How To's and list problems with the documentation in gethugothemes.com!

**Reply by somratpro** [reply]
Hey, cool idea. We can talk here about the docs feature and bugs

**Reply by kokunas** [reply]
Hi there,

First of all thanks for the AirSpace template, it looks cool! I'm building a site based on it.

I've been playing around as a newbie, and I have several proposals about the doc. For instance:

- The doc could provide several real examples and not only parameters, like how to setup the default language (before getting the proper `= "es"`, I tried `= es`, `= "spanish"`, `= "es-es"`, `= "ES"`)
- All pages inside de the code are `_index.md`, but the doc says `page.md`.
- I miss the Privacy/Cookies policies. Specially important after GDPR regulation.
Don't know if what I'm saying makes sense for you, or it is because my lack of knowledge. Hope this helps.

Kind regards,
kokuna

**Reply by kokunas** [reply]
Two more proposals:

* Fix the top menu, becase now you loose it while scrolling down.
* Include google maps embebed in contact page.

Kind regards,
kokuna
